macOS/DuckDuckGo/Common/View/SwiftUI/BannerView.swift

```diff
@@ -18,14 +18,15 @@
 
 import SwiftUI
 import SwiftUIExtensions
+import DesignResourcesKit
```

Comment:
I believe this import is not necessary given that we are injecting the theme manager.